src/operator/tensor/elemwise_unary_op.h

```diff
@@ -91,7 +92,11 @@ void IdentityComputeRsp(const nnvm::NodeAttrs& attrs,
   auto &output = outputs[0];
   CHECK_NE(req[0], kNullOp) << "kNullOp in IdentityComputeEx not supported yet";
   CHECK_NE(req[0], kWriteInplace) << "kWriteInplace in IdentityComputeEx not supported yet";
-  if (!input.storage_initialized()) return;
+  if (!input.storage_initialized()) {
+    NDArray out = output;
+    FillZerosRspImpl(s, &out);
+    return;
+  }
   TShape shape = input.aux_shape(rowsparse::kIdx);
```

Comment:
@cjolivier01 maybe you fixed it already but just in case you haven't. This may lead to test failure for binary/unary ops